.eslintrc.js

```diff
@@ -8,5 +8,23 @@ module.exports = {
     'ember/avoid-leaking-state-in-ember-objects': 'off',
 
     'ember-best-practices/require-dependent-keys': 'off',
+
+    'no-restricted-imports': [
+      'error',
+      {
+        paths: [
+          {
+            name: '@ember/object',
+            importNames: ['observer'],
+            message: 'For compatibility, import observers from -private/utils/observer',
```

Comment:
should it be "For compatibility reasons, use `import { addObserver, removeObserver } from '-private/utils/observer';`"?